tests/system/lab/test

```diff
@@ -0,0 +1,583 @@
+
+echo "======================= PI SETUP COMPLETE ======================="
+
+
+# Add Expression Filter (Square)
+curl -sX POST "$URL/filter" -d \
+'{
+   "name": "Square",
+   "plugin": "expression",
+   "filter_config": {
+      "name": "square",
+      "expression": "if(sinusoid>0,0.5,-0.5)",
+      "enable": "true"
+   }
+}'
+echo
+# Apply Square to Sine
+curl -sX PUT "$URL/filter/Sine/pipeline?allow_duplicates=true&append_filter=true" -d \
+'{
+   "pipeline": [
+      "Square"
+   ]
+}'
+echo
+echo "======================= SINUSOID SQUARE FILTER COMPLETE ======================="
+# Add Expression Filter (Max)
+curl -sX POST "$URL/filter" -d \
+'{
+   "name": "Max",
+   "plugin": "expression",
+   "filter_config": {
+      "name": "max",
+      "expression": "max(sinusoid, square)",
+      "enable": "true"
+   }
+}'
+echo
+# Apply Max to Sine
+curl -sX PUT "$URL/filter/Sine/pipeline?allow_duplicates=true&append_filter=true" -d \
+'{
+   "pipeline": [
+      "Max"
+   ]
+}'
+echo
+
+for LOOP in $(seq ${MAX_RETRIES}); do
+    RESULT=`curl -sX GET "$URL/asset/sinusoid?seconds=600"`
+    echo ${RESULT}
+    SQUARE=`echo ${RESULT} | jq '.[0].reading.square // empty'`
+    MAX=`echo ${RESULT} | jq '.[0].reading.max // empty'`
+    if [[ -n "$SQUARE" ]] && [[ -n "$MAX" ]]; then break; fi
+done
+if [[ ${LOOP} -eq ${MAX_RETRIES} ]]
+ then
+   display_and_collect_err "TIMEOUT! square and max data not seen in sinusoid graph. $URL/asset/sinusoid?seconds=600"
+   if [[ ${EXIT_EARLY} -eq 1 ]]; then exit 1; fi
+ else
+   echo "---- square and max data seen in sinusoid graph ----"
+fi
+# [
+#    {
+#       "reading": {
+#          "sinusoid": -0.809016994
+#          "square": -0.809016994
+#          "max": -0.809016994
+#       },
+#       "timestamp": "2019-08-28 07:55:12.810924"
+#    }, ...
+# ]]
+
+echo "======================= SINUSOID MAX FILTER COMPLETE ======================="
+
+
+# Add Randomwalk South
+curl -sX POST "$URL/service" -d \
+'{
+   "name": "Random",
+   "type": "south",
+   "plugin": "randomwalk",
+   "enabled": true,
+   "config": {}
+}'
+echo
+# need to wait for FogLAMP to be ready to accept python file
+sleep ${SLEEP_FIX}
+# Add Python35 Filter (ema)
+curl -sX POST "$URL/filter" -d \
+'{
+   "name": "Ema",
+   "plugin": "python35",
+   "filter_config": {
+      "config": {
+         "rate": 0.07
+      },
+      "enable": "true"
+   }
+}'
+echo
+# Apply Ema to Random
+curl -sX PUT "$URL/filter/Random/pipeline?allow_duplicates=true&append_filter=true" -d \
+'{
+   "pipeline": [
+      "Ema"
+   ]
+}'
+echo
+# Upload Ema python script
+curl -sX POST "$URL/category/Random_Ema/script/upload" -F "script=@scripts/ema.py"
+echo
+
+for LOOP in $(seq ${MAX_RETRIES}); do
+    RESULT=`curl -sX GET "$URL/asset/randomwalk?seconds=600"`
+    echo ${RESULT}
+    RANDOM=`echo ${RESULT} | jq '.[0].reading.randomwalk // empty'`
+    EMA=`echo ${RESULT} | jq '.[0].reading.ema // empty'`
+    if [[ -n "$RANDOM" ]] && [[ -n "$EMA" ]]; then break; fi
```

Comment:
We should not use reserved keywords here. So please change RANDOM to RANDOM_WALK or any meaningful word